policies/bundles/ResourceInventory/metric.rego

```diff
@@ -0,0 +1,13 @@
+package clouditor.metrics.resource_inventory
+
+default applicable = false
+
+default compliant = false
+
+applicable = true
+
+# all resources all applicable
+
+compliant = true
+
+# all resources are automatically compliant once we registered an evidence for them
```

Comment:
Thus, we argue that if you use the clouditor you have an inventory in place?

Reply:
Yup